### ISSUE TITLE
Rename to eventbrite

### ISSUE THREAD

**Issue by kmamykin** [therabidbanana/eventbrite#4]
The actual site this gem is integrating with is http://eventbrite.com , the gem is called eventbriGHT.

**Comment by therabidbanana** [comment]
Good point. I don't actually remember why exactly that is, but it's probably one of the following:
1. Since Eventbrite's api can't spell the word "address" right, I figured I'd return the favor.
2. I accidentally made the github repo and then was too lazy to rename it.
3. I left the "eventbrite" name open in the hope that someone would come along and make a better gem.

They all sound like plausible reasons, and the site I made this for is no longer being worked on, so I haven't really given this much attention in the last year - but it looks like I'm listed on EventBrite's page as the only Ruby library, so renaming it would probably be a good idea.
